person.rb

```diff
@@ -13,9 +18,20 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
+
+# person = Person.new(22, 'maximilianus')
+# person.correct_name
+# capitalizedPerson = CapitalizeDecorator.new(person)
+# capitalizedPerson.correct_name
+# capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
+# capitalizedTrimmedPerson.correct_name
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧